src/main/java/ru/javawebinar/topjava/model/MealWithExceed.java

```diff
@@ -20,8 +43,9 @@ public MealWithExceed(LocalDateTime dateTime, String description, int calories,
 
     @Override
     public String toString() {
-        return "UserMealWithExceed{" +
-                "dateTime=" + dateTime +
+        return "MealWithExceed{" +
+                "id=" + id +
+                ", dateTime=" + dateTime +
                 ", description='" + description + '\'' +
                 ", calories=" + calories +
                 ", exceed=" + exceed +
```

Comment:
Do you know hot keys to generate toString(), equals() and hashCode() in Intellij IDEA ?

Reply:
Yes. Alt+Ins